zoo-css-challenge/style.css

```
@@ -105,27 +107,61 @@ footer {
 
 .image-container img {
   vertical-align: top;
-  border: 5px solid #747c92;
 }
 
-.image-container img:first-of-type {
-  vertical-align: bottom;
+.bear1{
+  border: 5px solid #6ce946;
+}
+
+.bear2{
+  border: 5px solid #f80dc5;
+}
+
+.bear3{
+  border: 5px solid #0847f5;
 }
 
 .tigers {
   background-color: #0b3c49;
   color: #f0e100;
 }
 
+.tigers h2 {
+  color: #f8f0fb;
+}
+
 .giraffe {
-  background-color: #465775;
-  color: #56e39f;
+  background-color: #483C46;
+  color: #BEEE62;
+}
+
+.giraffeMain {
```

```suggestion
.giraffe-main {
```
CSS classes are typically in [kebab-case](https://winnercrespo.com/naming-conventions/#kebab-case), so we can change this to follow this convention